src/pages/[userId]/prospects.js

```diff
@@ -108,39 +76,41 @@ export default function Prospects() {
           <h4>Other Ministers</h4>
         </Grid>
       </Grid>
-      {prospects.map((prospect) => (
-        <Grid
-          container
-          spacing={1}
-          key={prospect.id}
-          className="striped columns"
-        >
-          <Grid item md={2} xs={3}>
-            <p>{path(['names', '0'], prospect)}</p>
-          </Grid>
-          <Grid item md={2} xs={3}>
-            <p>{prop('status', prospect)}</p>
-          </Grid>
-          <Grid item md={2} xs={3}>
-            <p>{prop('last_contacted', prospect)}</p>
-          </Grid>
-          <DesktopOnly>
-            <Grid item md={3}>
-              <p>{path(['notes', '0'], prospect)}</p>
+      {isLoading && <h1>Loading data...</h1>}
+      {prospects
+        && prospects.map((prospect) => (
+          <Grid
+            container
+            spacing={1}
+            key={prospect.id}
+            className="striped columns"
+          >
+            <Grid item md={2} xs={3}>
+              <p>{getPreferredName(prospect)}</p>
+            </Grid>
+            <Grid item md={2} xs={3}>
+              <p>{prop('status', prospect)}</p>
+            </Grid>
+            <Grid item md={2} xs={3}>
+              <p>{prop('last_contacted', prospect)}</p>
```

Comment:
Let's format this to only show month and year. Probably something like `Jan 1`
Look into https://date-fns.org/

Reply:
I formatted it to add the "nd, rd, th, or st" at the end of the day, but can easily take it off